src/cascadia/TerminalSettingsModel/ColorScheme.idl

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.Terminal.Settings.Model
 {
     [default_interface] runtimeclass ColorScheme : Windows.Foundation.IStringable {
+        ColorScheme();

```

Comment:
In this case I agree that it might be a little weird, the problem was just that `IInheritableSetting` required an empty constructor so I just followed the compiler here.